module/solutions/helper.js

```diff
@@ -1470,6 +1470,52 @@ module.exports = class SolutionsHelper {
     })
    } 
 
+    /**
+   * List solutions.
+   * @method  
+   * @name listByProgramId
+   * @param {String} programId - Program id.
+   * @returns {JSON} List of solutions.
+   */
+  
+  static listByProgramId(programId) {
```

Comment:
@aman-tunerlabs Not required as discussed.

Reply:
@aks30  Resolved